scripts/van_allen_tools/migrate_van_allen_workspaces.py

```diff
@@ -392,8 +410,14 @@ def setup_single_workspace(workspace, ignore=[]):
     if not get_ad_success:
         return workspace_dict
 
+    # get workspace attributes including workspace data from source workspace to destination workspace
+    get_workspace_success, get_workspace_message = get_workspace_attributes(source_workspace_namespace, source_workspace_name)
```

Comment:
I know the comment describes it, however I think just the "get workspace" is something we actually do in one of the other steps to confirm if a workspace with the same name already exists. I know it would be really long as a name for a variable but could we call these `get_ws_attributes_success` and get_ws_attributes_message` or something like that so we can know for sure what step we are doing and checking?